File: C++/05-Operadores Relacionais/Relacionais.cpp
#include<iostream>
#include<stdlib.h>

using namespace std;

int main()
{
    //Operadores relacionais testam uma relação entre variaveis(se são verdadeiras ou falsas)
    //Se for verdadeira retorna 1 se for falsa retorna 0.
    int x = 4;
    int y = 3;
    int z = 3;
    bool teste;

    cout << "Operadores Relacionais" << endl;
    cout << "1- Verdadeiro / 0- Falso" << endl;

    //os operadores relacionais em C+= são:
    //< <= > >= == !=
    teste = x > y;
    cout << x << " > [maior] " << y << " ? " << teste <<endl; //verdadeiro(1)
    teste = x < y;
    cout << x << " < [menor] " << y << " ? " << teste <<endl; //falso(0)
    teste = y >= z;
    cout << y << " >= [maior ou igual] " << z << " ? " << teste <<endl; //verdadeiro(1)
    teste = x <= z;
    cout << x << " <= [menor ou igual] " << z << " ? " << teste <<endl; //falso(0)
    teste = x == y;
    cout << x << " == [igual] " << y << " ? " << teste <<endl; //falso(0)
    teste = x != z;
    cout << x << " != [diferente] " << z << " ? " << teste <<endl; //verdadeiro(1)

    system("pause");
    return 0;
}

File: Python/05-Operadores Relacionais/Relacionais.py
# Operadores reacionais testam a relação entre duas variaveis

x = 4
y = 2
z = 7
w = 2

#Os operadores relacionais em Python são :
# Maior >, Menor <
# Maior ou igual >=, Menor ou igual <=
# Igual ==

print("{0} > [maior que] {1} ? {2}".format(x, y, x > y)) # Verdadeiro
print("{0} < [menor que] {1} ? {2}".format(z, y, x < y)) # Falso
print("{0} >= [maior ou igual que] {1} ? {2}".format(x, z, x >= z)) # Falso
print("{0} <= [menor ou igual que] {1} ? {2}".format(y, w, y <= w)) # Verdadeiro
print("{0} == [igual a] {1} ? {2}".format(y, z, y == z)) # Falso


